src/main/java/org/threadly/litesockets/SocketExecuterCommonBase.java

```diff
@@ -145,16 +144,13 @@ public void stopListening(final Server server) {
     if(!checkServer(server)) {
       return;
     } else {
-      if(server.getServerType() == WireProtocol.TCP) {
+      if(server instanceof TCPServer) {
```

Comment:
its more correct esp with the added casting below, and honestly the performance difference is pretty minimal.  This is also only ever called if you call .stop() on a server instance so I dont see a practical use case where performance is gonna matter a lot there.